src/js/components/devices/expanded-device.js

```diff
@@ -198,6 +239,11 @@ export const ExpandedDevice = ({
           <Button to={`/deployments?open=true&deviceId=${device.id}`} component={ForwardingLink} startIcon={<ReplayIcon />}>
             Create a deployment for this device
           </Button>
+          {!isGateway && (
+            <Button onClick={onMakeGatewayClick} startIcon={<GatewayIcon className={classes.gatewayIcon} />} style={{ marginLeft: 30 }}>
+              Promote to Gateway
```

Comment:
To be consistent with the taxonomy from `MakeGatewayDialog`,

```suggestion
              Promote to gateway
```

or

```suggestion
              Promote to Mender Gateway
```

Reply:
Thanks - adjusted!